src/texts.js

```diff
@@ -77,3 +50,32 @@ export function getText(lang, name, placeholders = {}) {
     text,
   )
 }
+
+let DATA = null;
+
+export async function loadTexts(lang = LANG.SK) {
+  const promises = [
+    {file: 'end_of_orders.txt', name: TEXTS.END_OF_ORDERS},
+    {file: 'food_arrived.txt', name: TEXTS.FOOD_ARRIVED},
+    {file: 'food_wont_arrive.txt', name: TEXTS.FOOD_WILL_NOT_ARRIVE},
+    {file: 'greetings_new.txt', name: TEXTS.GREETING_NEW},
+    {file: 'greetings_saved.txt', name: TEXTS.GREETING_SAVED},
+    {file: 'help.txt', name: TEXTS.HELP_COMMANDS},
+    {file: 'last_call.txt', name: TEXTS.LAST_CALL},
+    {file: 'load_failed.txt', name: TEXTS.MENU_LOAD_FAILED},
+    {file: 'no_dm.txt', name: TEXTS.NO_DM},
+    {file: 'unknown_order.txt', name: TEXTS.UNKNOWN_ORDER},
+    {file: 'your_food_arrived.txt', name: TEXTS.YOUR_FOOD_ARRIVED},
+    {file: 'your_food_wont_arrive.txt', name: TEXTS.YOUR_FOOD_WILL_NOT_ARRIVE},
+  ].map(({file, name}) => loadTextAsync(PATH_TO_MESSAGE_FILES + lang + '/' + file, lang, name))
```

Comment:
Use `path.join()` instead pls

Reply:
rewritten to [PATH_TO_MESSAGE_FILES, lang, file].join('/')

Reply:
I understand now. I didn't know such function exists. :man_facepalming: